utils/obsproc/Ghrsst2Ioda.h

```diff
@@ -57,64 +68,51 @@ namespace gdasapp {
       }
 
       // datetime: Read Reference Time
-      std::vector<int> refTime(dimTime);
+      std::vector<int32_t> refTime(dimTime);
       ncFile.getVar("time").getVar(refTime.data());
       std::string refDate;
       ncFile.getVar("time").getAtt("units").getValues(refDate);
 
       // Read sst_dtime to add to the reference time
-      int sstdTime[dimTime][dimLat][dimLon];  // NOLINT
-      ncFile.getVar("sst_dtime").getVar(sstdTime);
-      float dtimeOffSet;
-      ncFile.getVar("sst_dtime").getAtt("add_offset").getValues(&dtimeOffSet);
+      // TODO(AMG): What's below does not read the field the same way python does
+      std::vector<int32_t> sstdTime(dimTime*dimLat*dimLon);
```

Comment:
Minor style thing - is `dimTime*dimLat*dimLon` calculated elsewhere as `numloc` or some such?

Reply:
nope.

Reply:
nope.